clouddatabasesv5/cloud_databases_v5.go

```diff
@@ -4188,6 +4196,13 @@ const (
 	GetDefaultScalingGroupsOptionsTypePostgresqlConst = "postgresql"
 )
 
+// Constants associated with the GetDefaultScalingGroupsOptions.HostFlavor property.
+// When a host_flavor of 'multitenant' is included with the request, IBM Cloud Database's new shared compute default
+// groups will be returned.
+const (
+	GetDefaultScalingGroupsOptionsHostFlavorMultitenantConst = "multitenant"
```

Comment:
Was this auto generated? Anybody using it?

Reply:
Yes it was auto-generated. We're not using it though.